src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -28,9 +28,10 @@ struct event_entry_t {
   uint32_t time_to_live = DEFAULT_GLOBAL_VALUE;
   uint32_t max_retries = DEFAULT_GLOBAL_VALUE;
   uint32_t retry_sleep_duration = DEFAULT_GLOBAL_VALUE;
+  bool needs_migration = false;
```

Comment:
why not init to "true"?

Reply:
I was doing it in the else bracket if it didn't decode, but this way is optimal